dataplane/src/args.rs

```diff
@@ -52,21 +52,24 @@ pub(crate) struct CmdArgs {
 
     #[arg(
         long,
-        value_name = "Unix socket for FRR to send messages to the dataplane control plane interface",
+        value_name = "CPI Unix socket path",
+        help = "Unix socket for FRR to send route update messages to the dataplane",
         default_value = DEFAULT_DP_UX_PATH
     )]
     cpi_sock_path: String,
 
     #[arg(
         long,
-        value_name = "Unix socket to listen for dataplane cli connections",
+        value_name = "CLI Unix socket path",
+        help = "Unix socket to listen for dataplane cli connections",
         default_value = DEFAULT_DP_UX_PATH_CLI
     )]
     cli_sock_path: String,
 
     #[arg(
         long,
-        value_name = "Unix socket to connect to FRR agent that controls FRR configuration reload",
+        value_name = "FRR Agent Unix socket path",
+        help = "Unix socket to connect to FRR agent that controls FRR configuration reload",
```

Comment:
I would be more precise on the descriptions, esp. in the last. The frr-agent socket is the remote address of the frr-agent, unlike the first two, which are addresses of the dataplane process. I'd also squash this commit with the first?

Reply:
So, I'd say:
   help = "Address of unix socket of frr-agent" (or where frr-agent listens for connections)